### PR TITLE
Re-enable desktop tests

### PR DESCRIPTION
**Pull request by pranavkm**
* Cleanup tests to use a test case per scenario
* Run tests on CLR
* Enable view precompilaton for the sample.

### CODE REVIEW

File: samples/MusicStore/MusicStore.csproj [hunk 1]
```diff
@@ -4,8 +4,9 @@
 
   <PropertyGroup>
     <Description>Music store application on ASP.NET Core</Description>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
```
Comment (pranavkm) [inline samples/MusicStore/MusicStore.csproj]: @Eilon - do we care about porting this to preview2?

Comment (Eilon) [inline samples/MusicStore/MusicStore.csproj]: Naah.

**Comment by pranavkm** [comment]
Well, I ran it about a couple dozen times and it ran without failures. It takes fairly long to run one round of functional tests in this repo.

**Comment by pranavkm** [comment]
@kichalla I've started getting failures with Google auth as part of the OpenId and SmokeTests. Any idea what's up?

**Pull request closed without merging**